tools/ci_build/github/azure-pipelines/android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml

```
@@ -0,0 +1,94 @@
+jobs:
+- job: Build_QNN_EP
+  pool: Onnxruntime-QNNEP-Ubuntu-2004-CPU
+  timeoutInMinutes: 30
+  workspace:
+    clean: all
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: Use Python $(pythonVersion)
+    inputs:
+      versionSpec: $(pythonVersion)
+
+  - script: sudo apt-get update -y && sudo apt-get install -y coreutils ninja-build
+    displayName: Install coreutils and ninja
+
+  - script: /bin/bash tools/ci_build/github/android/setup_gradle_wrapper.sh $(pwd)
+    displayName: Setup gradle wrapper to use gradle 6.8.3
```

This file no longer exists. It was removed by https://github.com/microsoft/onnxruntime/pull/14733
We now have to use a template.

```suggestion
  - template: templates/set-up-gradle-wrapper-step.yml
```